libs/shared/ui/src/lib/components/modal/modal-root.tsx

```diff
@@ -48,13 +50,18 @@ interface ModalProviderProps {
 export const ModalProvider = (props: ModalProviderProps) => {
   const [openModal, setOpenModal] = useState(false)
   const [contentModal, setContentModal] = useState(<></>)
-  const [optionsModal, setOptionsModal] = useState({
-    width: 488,
-  })
+  const [optionsModal, setOptionsModal] = useState({})
   const [alertClickOutside, enableAlertClickOutside] = useState(false)
   const [modalAlertOpen, setModalAlertOpen] = useState(false)
   const [alertModalChoice, setAlertModalChoice] = useState<boolean | undefined>(undefined)
 
+  // Reset modal options when the modal is closed
+  useEffect(() => {
+    if (!openModal) {
+      setOptionsModal(defaultContext.optionsModal)
+    }
+  }, [openModal])
```

Comment:
Here we need to reset the context options for Modal during the closing, useEffect is necessary to detect if modal is closed